test/integration/operator/terraform/argocd.tf

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
+resource "helm_release" "argo_rollouts" {
+  namespace        = "argo"
+  create_namespace = true
+  name             = "argo-rollouts"
+  repository       = "https://argoproj.github.io/argo-helm"
+  chart            = "argo-rollouts"
+}
```

Comment:
We may want to specify a version for this chart so it's pinned for reproducibility?